charts/secrets-store-csi-driver/values.yaml

```diff
@@ -5,6 +5,10 @@ linux:
     tag: v1.4.3
     #digest: sha256:
     pullPolicy: IfNotPresent
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+  # An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything.
+  tolerations:
```

Comment:
For backward compatibility should this default to linux.tolerations?  Either solution works for me, but it depends on how you hold backward compat.

Reply:
That's makes sense.
I'll fix it